front/src/common/api/generatedEditoastApi.ts

```diff
@@ -1508,6 +1577,26 @@ export type PostWorkSchedulesProjectPathApiArg = {
     work_schedule_group_id: number;
   };
 };
+export type BuiltinRole =
```

Comment:
`Write` means also update/delete or it will come later ?

Reply:
Yes `Write` = `create + update + delete`